### ISSUE TITLE
Radial in/out are swapped

### ISSUE THREAD

**Issue by JonnyOThan** [MOARdV/AvionicsSystems#355]
Clicking an ASET radial-out button actually points radial-in and vice versa.

This is an unfortunate consequence of the stock game getting this wrong.  I fixed this in RPM by doing the swap internally, so that all the content actually made sense: https://github.com/JonnyOThan/RasterPropMonitor/commit/618fc314be60b25f2278688fd2fbdad7e8738cd4

**Comment by MOARdV** [comment]
Yeah, I left comments in the code where I convert between KSP values and MAS values.

**Comment by JonnyOThan** [comment]
Hmm, if you swap it there..then why are the props still backwards?  Needs some investigation I guess.
